cli.js

```diff
@@ -44,6 +45,14 @@ if (semver.lt(process.version, '5.0.0')) {
   process.exit(1);
 }
 
+if (typeof cli.flags.mobile === 'undefined') {
```

Comment:
do we want to set default values here or in runner.js? We can defer on this since not a priority right now, but setting it here makes it a required option for anyone running us